model/profile.go

```diff
@@ -58,7 +58,14 @@ func (pp PprofProfile) Transform(ctx context.Context, cfg *transform.Config) []b
 	valueFieldNames := make([]string, len(pp.Profile.SampleType))
 	for i, sampleType := range pp.Profile.SampleType {
 		sampleUnit := normalizeUnit(sampleType.Unit)
-		valueFieldNames[i] = sampleType.Type + "." + sampleUnit
+		// Go profiles report samples.count, Node.js profiles report sample.count.
+		// We use samples.count for both so we can aggregate on one field.
```

Comment:
Convention says that the first sample type is always the number of samples collected. However, for heap profiles this field is `alloc_objects.count`, which is explicitly aggregated on in fetcher.go (and passed to the pprof UI).